libevmasm/Assembly.cpp

```diff
@@ -217,65 +218,74 @@ string Assembly::toStringInHex(u256 _value)
 	return hexStr.str();
 }
 
-Json::Value Assembly::assemblyJSON(StringMap const& _sourceCodes) const
+Json::Value Assembly::assemblyJSON(map<string, unsigned> const& _indices) const
 {
 	Json::Value root;
 
 	Json::Value& collection = root[".code"] = Json::arrayValue;
 	for (AssemblyItem const& i: m_items)
 	{
+	    auto iter = _indices.find(i.location().source->name());
+	    int index = iter != _indices.end() ? iter->second : -1;
```

Comment:
```suggestion
	    int index = iter == _indices.end() ? -1 : iter->second;
```
(code with fewer negations is usually better to understand)

Reply:
Also maybe `sourceIndex`?

Reply:
Fixed as recommended.
Renamed index to sourceIndex as well as it is more descriptive name